roles/libwww/vars/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # vars file for roles/libwww
-php_version: "7.2"
+php_version: "8.0"
```

Comment:
We have php_version: "8.0" both here and in group_vars. Do we need it in both places?

Reply:
it will fail CI if you don't add it there.